test/language/module-code/early-dup-top-function.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 10.2.1
```

Comment:
Two issues here: 

1. `es6id` is no longer used in new tests, please use `esid: ...` format
2. The section at 10.2.1 is "10.2.1 [[Call]] ( thisArgument, argumentsList )".

Reply:
I believe the correct information is: `esid: sec-module-semantics-static-semantics-early-errors`

Reply:
Thanks for the suggestions! Updated.